internal/postprocessor/main.go

```diff
@@ -280,23 +341,24 @@ func docURL(cloudDir, importPath string) (string, error) {
 	return "https://cloud.google.com/go/docs/reference/" + mod + "/latest/" + pkgPath, nil
 }
 
-func (c *config) AmendPRDescription(ctx context.Context) error {
+func (c *config) AmendPRDescription(ctx context.Context) ([]string, error) {
 	log.Println("Amending PR title and body")
 	pr, err := c.getPR(ctx)
 	if err != nil {
-		return err
+		return []string{}, err
 	}
-	newPRTitle, newPRBody, err := c.processCommit(*pr.Title, *pr.Body)
+	newPRTitle, newPRBody, modules, err := c.processCommit(*pr.Title, *pr.Body)
 	if err != nil {
-		return err
+		return []string{}, err
 	}
-	return c.writePRCommitToFile(newPRTitle, newPRBody)
+	return modules, c.writePRCommitToFile(newPRTitle, newPRBody)
 }
 
-func (c *config) processCommit(title, body string) (string, string, error) {
+func (c *config) processCommit(title, body string) (string, string, []string, error) {
```

Comment:
I think this functions is starting to return too much. I would either refactor this work to be in multiple functions if possible or return a struct of data.

Reply:
I've changed the logic so that, rather than returning scopes, it simply modifies the config within the function. Do you think this return is okay now?